### PR TITLE
Added BylineImage style and story

### PR DESCRIPTION
**Pull request by ThomasLilja**

### CODE REVIEW

File: stories/editorialMarking/bylineImage.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	SiteSelection,
+} from '../storybook-components';
+
+import LeetTextarea from '../../src/atoms/LeetTextarea';
+
+import Image from '../../src/atoms/Image';
+import BylineImage from '../../src/atoms/BylineImage';
+
+export default () => (
+	<SiteSelection>
```
Comment (misund) [inline stories/editorialMarking/bylineImage.js]: The SiteSelection is included [here](https://github.com/dbmedialab/shiny/pull/30/files#diff-156d33391f29ea7da619ba648e042ea0R14) already, this one can be removed

File: src/atoms/BylineImage.js [hunk 2]
```diff
@@ -1,6 +1,14 @@
 import styled from 'styled-components';
 
 const BylineImage = styled.img`
+	width: 6.4rem;
+   height: 6.4rem;
```
Comment (misund) [inline src/atoms/BylineImage.js]: Is something weird about this indentation?

Comment (ThomasLilja) [reply src/atoms/BylineImage.js]: Yes, there was something weird. It looked good in my editor but it is fixed now.

File: stories/editorialMarking/bylineImage.js (hunk 1 above)
Comment (ThomasLilja) [reply stories/editorialMarking/bylineImage.js]: Removed.

**Pull request merged**